### PR TITLE
[tests-only] code cleanup

### PR DESCRIPTION
**Pull request by individual-it**
- delete unused functions
- standardise names of interaction functions
- clarify what the variables `username` and `password` are used for by renaming to `adminUsername` and `adminPassword`
- change all occurrence of hard-coded admin username to use variable from config file

### CODE REVIEW

File: tests/unauthorized/userTest.js [hunk 1]
```diff
@@ -8,15 +8,15 @@ describe('Unauthorized: Currently testing user management,', function () {
...
+    getCapabilitiesWithInvalidAuthInteraction,
     createOwncloud
   } = require('../pactHelper.js')
   const {
     unauthorizedXmlResponseBody,
     createProvider
   } = require('../pactHelper.js')
 
-  const addInvalidAuthInteraction = (provider, requestName, method, path, query) => {
+  const nvalidAuthInteraction = (provider, requestName, method, path, query) => {
```
Comment (saw-jan) [inline tests/unauthorized/userTest.js:19]:
```suggestion
  const InvalidAuthInteraction = (provider, requestName, method, path, query) => {
```
I think it is supposed to be `InvalidAuthInteraction` ( both pronounce similar though :smile: )

Comment (individual-it) [reply tests/unauthorized/userTest.js]: good catch, thanks
I've changed it

File: tests/userTest.js [hunk 2]
```diff
@@ -6,17 +6,17 @@ describe('Main: Currently testing user management,', function () {
...
-    capabilitiesGETRequestValidAuth,
-    GETRequestToCloudUserEndpoint,
-    createAUser,
-    deleteAUser,
-    createAUserWithGroupMembership,
+    getCapabilitiesInteraction,
+    getCurrentUserInformationInteraction,
+    createAUserInteraction,
+    deleteUserInteraction,
+    createAUserWithGroupMembershipInteraction,
```
Comment (phil-davis) [inline tests/userTest.js:13]:
```suggestion
    createUserInteraction,
    deleteUserInteraction,
    createUserWithGroupMembershipInteraction,
```
You removed "A" from `deleteAUser` - could do the same for the `createAUser*`

**Review by phil-davis — COMMENTED** [review]
Looks good. CI dies for some reason.

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/owncloud-sdk/1608/3/11 fails.
It needs some more love.

**Comment by individual-it** [comment]
increasing the log-level seems to make it pass, strange

**Pull request merged**